PATCHES.md

```diff
@@ -18,32 +18,49 @@ This is an overview over all patches that are currently used.
 | server |  Add config Yatopia command and basic settings      | tr7zw |  |
 | api |  Add getLastTickMs api      | tr7zw |  |
 | server |  Add getLastTickMs api      | tr7zw |  |
+| server |  Add more dupe options      | Mariell Hoversholm |  |
 | server |  Add option to disable saving projectiles to disk      | William Blake Galbreath |  |
 | server |  Add permission for F3+N debug      | William Blake Galbreath |  |
 | server |  Akarin Updated Save json list async      | tsao chi |  |
+| server |  Akarin updated Async Path Finding      | tsao chi | Bud Gidiere |
```

Comment:
WTF?! WHY IS THIS STILL PORTING 

@budgidiere please fix scripts kthx untill then we're not merging this

Reply:
Wait nvm it looks like its adding it automatically to patches.md, it doesn't seem to be applying anything